.github/workflows/docker.yaml

```diff
@@ -323,36 +345,59 @@ jobs:
           EOF
           gcloud auth activate-service-account --key-file .github/gcloud-auth.json
           gcloud auth configure-docker --quiet
+          gcloud auth configure-docker us-docker.pkg.dev --quiet
       - name: Get canonical image name
         id: name
         run: echo "canonical=$(echo ${{ needs.image.outputs.images }} | sed 's/ .*//')" >> "$GITHUB_OUTPUT"
       - name: Create and push manifest
         run: |
           set -x
-          IMGS=$(echo ${{ inputs.archs }} | tr ' ' '\n' | sed 's%^%${{ steps.name.outputs.canonical }}-%' | tr '\n' ' ')
-          # Sometimes GCR gives random errors. Let's try a few times before giving up.
-          for RETRY in $(seq 10) ; do
-            for ARCH in ${{ inputs.archs }} ; do
-              if ! docker pull "${{ steps.name.outputs.canonical }}-${ARCH}" ; then
-                echo "Attempt #${RETRY} failed."
-                sleep 10
-                # Continue the outer loop, not the inner one.
-                continue 2
+          # Will look like "gcr.io/ironcore-dev-1/ironcore-id-server:1.2.6"
+          CANONICAL="${{ steps.name.outputs.canonical }}"
+          # Will look like "gcr.io/ironcore-dev-1/ironcore-id-server:1.2.6-amd64 gcr.io/ironcore-dev-1/ironcore-id-server:1.2.6-arm64"
+          IMGS=$(echo ${{ inputs.archs }} | tr ' ' '\n' | sed "s%^%$CANONICAL-%" | tr '\n' ' ')
+          test_manifest_creation() {
+            local target="$1"
+            for RETRY in $(seq 10); do
+              echo "Attempt #$RETRY to pull images and create manifest for $target"
+              for ARCH in ${{ inputs.archs }} ; do
+                if ! docker pull "${CANONICAL}-${ARCH}" ; then
+                  echo "Attempt #${RETRY} failed."
+                  sleep 10
+                  # Continue the outer loop, not the inner one.
+                  continue 2
+                fi
+              done
+              if docker manifest create "$target" ${IMGS} ; then
```

Comment:
I hadn't noticed this before, but creating the manifest, then deleting it just to recreate it below seems a bit odd? If we were able to pull the above images, how could manifest creation fail?